kuma/settings/common.py

```diff
@@ -1847,7 +1847,8 @@ def get_user_url(user):
 # Whether or not to enable the BCD signalling feature.
 # Affects loading of CSS (statically) and JS (in runtime).
 ENABLE_BCD_SIGNAL = config('ENABLE_BCD_SIGNAL', default=True, cast=bool)
-# Multi Auth Switch
+# Enable or disable the multi auth(Google and Github) sign-in flow
+# When disabled, Github will be the default and only Auth provider
 MULTI_AUTH_ENABLED = config('MULTI_AUTH_ENABLED', default=False, cast=bool)
```

Comment:
Too late now but it would be nice to have the same pattern for naming these kinds of Boolean constants. 
So, this should be ENABLE_MULTI_AUTH